vault/src/issue.rs

```diff
@@ -201,11 +203,21 @@ async fn process_transaction_and_execute_issue<B: BitcoinCoreApi + Clone + Send
                 let proof = btc_rpc.get_proof_for(txid.clone(), &block_hash).await?;
 
                 info!("Executing issue with id {}", issue_id);
-
-                // this will error if someone else executes the issue first
-                provider
+                match provider
                     .execute_issue(issue_id, H256Le::from_bytes_le(&txid.as_hash()), proof, raw_tx)
-                    .await?;
+                    .await
+                {
+                    Ok(_) => (),
+                    Err(RuntimeError::XtError(SubxtError::Runtime(SubxtRuntimeError::Module(SubxtModuleError {
+                        ref module,
+                        ref error,
+                    }))))
+                        if module == ISSUE_MODULE && error == ISSUE_COMPLETED_ERROR =>
+                    {
+                        info!("Issue {} has already been completed", issue_id);
```

Comment:
Maybe we should log as debug? Just to reduce the number of questions we get about this

Reply:
I think it makes sense to log as info because it's the same workaround as in the relay submission, operators should know that a transaction was submitted.